modules/caddytls/matchers.go

```diff
@@ -21,10 +21,10 @@ import (
 	"net/netip"
 	"strings"
 
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
 	"github.com/caddyserver/certmagic"
 	"go.uber.org/zap"
```

Comment:
```suggestion
 	"github.com/caddyserver/certmagic"
 	"go.uber.org/zap"

	"github.com/caddyserver/caddy/v2"
	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
```

I think it has to be this way. Basically the same-repo imports grouped at the end. Stdlib first, any other deps second, same-repo third.

Reply:
Thank you for a detailed explanation!